To-do list/Controllers/LoginViewController.swift

```diff
@@ -31,76 +26,19 @@ class LoginViewController: UIViewController {
             Alert.showBasic(
                 title: "Please enter your password",
                 vc: self)
-        } else if users.isEmpty  {
-            Alert.showBasic(
-                title: "Invalid login, please register",
-                vc: self)
         } else {
-            for i in users.indices {
-                if users[i].login == loginTextField.text, users[i].password == passwordTextField.text {
-                    userIndex = i
-                    let taskListVC = self.storyboard?.instantiateViewController(
-                        withIdentifier: "TaskListViewController") as? TaskListViewController
-                    taskListVC?.userIndex = userIndex
-                    self.navigationController?.pushViewController(taskListVC!, animated: true)
-                    
-                    loginTextField.text = ""
-                    passwordTextField.text = ""
-                    break
-                    
-                } else if users[i].login == loginTextField.text, users[i].password != passwordTextField.text {
-                    Alert.showBasic(
-                        title: "Invalid password",
-                        vc: self)
-                } else if users[i].login != loginTextField.text, i == users.count-1 {
-                    Alert.showBasic(
-                        title: "Invalid login, please register or correct login",
-                        vc: self)
-                }
-            }
+            
+            let taskListVC = self.storyboard?.instantiateViewController(
+                withIdentifier: "TaskListViewController") as? TaskListViewController
+            self.navigationController?.pushViewController(taskListVC!, animated: true)
```

Comment:
Create an use convenient extension of UIViewController to load Controller from Storyboard (using it's type).

Reply:
Extension UIViewController to load Controller from Storyboard is done and applied. Following code:

extension UIViewController {
    class func loadFromStoryboard<T: UIViewController>(type: T.Type) -> T {
        let storyboard = UIStoryboard(name: "Main", bundle: nil)
        let identifier = String(describing: T.self)
        return storyboard.instantiateViewController(withIdentifier: identifier) as! T
    }
}

**application:** 
            let taskListVC = TaskListViewController.loadFromStoryboard(type: TaskListViewController.self)
            self.navigationController?.pushViewController(taskListVC, animated: true)

In this case, we can access to the properties and methods on the next VC. It works:)